src/platforms/react-native/performance/included-instrumentation.mdx

```diff
@@ -134,22 +134,34 @@ class App extends React.Component {
   // Create a ref for the navigation container
   navigation = React.createRef();
 
-  componentDidMount() {
-    // Register the navigation container with the instrumentation
-    routingInstrumentation.registerNavigationContainer(navigation);
-  }
-
   render() {
     return (
       // Connect the ref to the navigation container
-      <NavigationContainer ref={this.navigation}>
+      <NavigationContainer
+        ref={this.navigation}
+        onReady={() => {
+          // Register the navigation container with the instrumentation
+          routingInstrumentation.registerNavigationContainer(navigation);
+        }}>
```

Comment:
do these changes require a migration page too? cus it's different and people won't figure it out if we don't tell them to "migrate" and what to "migrate"

Reply:
Makes sense, as this solves a bug when using it with the `linking` prop that I expect a lot of people to use. Maybe a small section on this page should suffice?

Reply:
I'd add it to https://docs.sentry.io/platforms/react-native/migration/

eg https://docs.sentry.io/platforms/android/migration/#migrating-from-iosentrysentry-410-to-iosentrysentry-420
migration from a minor bump is also fine